src/limiters/limiters.jl

```diff
@@ -0,0 +1,52 @@
+
+"""
+    PositivityPreservingLimiterZhangShu(; threshold, variables)
+
+The fully-discrete positivity-preserving limiter of
+- Zhang, Shu (2011)
+  Maximum-principle-satisfying and positivity-preserving high-order schemes
+  for conservation laws: survey and new developments
+  [doi: 10.1098/rspa.2011.0153](https://doi.org/10.1098/rspa.2011.0153)
+The limiter is applied to all scalar `variables` in their given order
```

Comment:
Can we add here a comment, that the order of the variables have and thus the application of the PP limiter might have a strong influence on the robustness.

Reply:
Good suggestion. Added in d86bd24.